extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -901,13 +900,15 @@ describes.realWin('amp-story', {
 
       return story.layoutCallback()
           .then(() => {
-            const qsStub = sandbox.stub(story.element, 'querySelectorAll');
-            qsStub.withArgs('amp-audio, [background-audio]').returns(['el']);
-            qsStub.withArgs('amp-video').returns(['el', 'el']);
+            const ampVideoEl = win.document.createElement('amp-video');
+            const ampAudoEl = createElementWithAttributes(win.document,
+                'amp-audio', {'background-audio': ''});
+            story.element.appendChild(ampVideoEl);
+            story.element.appendChild(ampAudoEl);
 
             const expected = {
               [MediaType.AUDIO]: 3,
-              [MediaType.VIDEO]: 4,
+              [MediaType.VIDEO]: 3,
```

Comment:
I'm not sure I understand what's going on here, but why does the expected number of max videos in the pool decrease with this change?

Reply:
Just because the test was faking that there were two `amp-video` elements, and I only appended one to make the test look cleaner. But let me know if you think it's better to keep as it was and append two `amp-video`s.

Reply:
Got it.  Thanks for the explanation.  Agree this makes it easier to read.